songs/die-aerzte/die-perfekte-diktatur.tex

```diff
@@ -0,0 +1,96 @@
+\beginsong{Die perfekte Diktatur}[by={Die Ärzte}]
+
+    \textnote{Your guitar should be tuned to an E$\,\flat$}
```

Comment:
Side note: I think one wouldn't drop the tuning of the guitar while playing songs from this songbook with friends or at camp fire. But we should still leave this comment as recommendation.

@dlik is the text note clear for you or should there be more info?

Reply:
How about we just write down the full tuning? Like this:

` \textnote{Tuning: E$\,\flat$ A$\,\flat$ D$\,\flat$ G$\,\flat$ H$\,\flat$ E$\,\flat$}`

Reply:
Or alternatively:

Reply:
> How about we just write down the full tuning? Like this:
> 
> ` \textnote{Tuning: E$\,\flat$ A$\,\flat$ D$\,\flat$ G$\,\flat$ B$\,\flat$ E$\,\flat$}`

Yeah I'm in favor of this